### PR TITLE
Allow users with same email domain on open-to-selected briefs

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

File: app/main/helpers/briefs.py [hunk 1]
```diff
@@ -23,12 +23,22 @@ def get_brief(data_api_client, brief_id, allowed_statuses=None):
 
 
 def is_supplier_selected_for_brief(data_api_client, current_user, brief):
+    def domain(email):
+        return email.split('@')[-1]
+
+    current_user_domain = domain(current_user.email_address) \
+        if domain(current_user.email_address) not in ['gmail.com', 'bigpond.com', 'outlook.com', 'hotmail.com'] \
```
Comment (matt-sm) [inline app/main/helpers/briefs.py]: should this be in config?

Comment (matt-sm) [inline app/main/helpers/briefs.py]: and I'm confused why your're checking this particular list?

File: app/main/helpers/briefs.py [hunk 2]
```diff
@@ -23,12 +23,22 @@ def get_brief(data_api_client, brief_id, allowed_statuses=None):
...
+        else None
+
     if brief.get('sellerSelector', '') == 'allSellers':
         return True
     if brief.get('sellerSelector', '') == 'someSellers':
-        return current_user.email_address in brief['sellerEmailList']
+        seller_domain_list = [domain(x) for x in brief['sellerEmailList']]
+        return current_user.email_address in brief['sellerEmailList']\
+            or current_user.email_address in seller_domain_list
```
Comment (maxious) [inline app/main/helpers/briefs.py]: Yep, will fix.

File: app/main/helpers/briefs.py (hunk 1 above)
Comment (maxious) [inline app/main/helpers/briefs.py]: Probably. We're making sure that this feature doesn't work for generic email hosting.

**Pull request merged**